conf/messages.en

```diff
@@ -1252,6 +1252,13 @@ transport.transportMeans.borderMode.subheading = Border mode of transport
 transport.transportMeans.borderMeans.subheading = Border means of transport
 transport.transportMeans.borderMeans.addOrRemove = Add or remove border means of transport
 
```

Comment:
Could you add a line in here to split the sections up - something like: # Supply chain actors

Reply:
Is that necessary seeing as it is all under the trader details section?

Reply:
Yea true, it's still a sub part of transport. Ignore this